CONTRIBUTORS.md

```diff
@@ -0,0 +1,20 @@
+# Contributors
+
+The ODL maintainers are grateful to acknowledge contributions from several people. What follows is an incomplete list, see [contributors](https://github.com/odlgroup/odl/graphs/contributors) for a complete list and feel free to add yourself!
+
+## Contributors in alphabetic order
+
+* [Jonas Adler](https://github.com/adler-j). Package maintainer. Created much of the `Set`, `LinearSpace` and `Operator` structure including utilities. Has also contributed to most other submodules.
+* [Chong Chen](https://github.com/chongchenmath). Started work on the `odl.deform` package.
+* [Matthias J. Ehrhardt](https://github.com/matthiasje). Several contributions to `odl.solvers`, in addition to general bug fixes and improvements.
+* [Barbara Gris](https://github.com/bgris). Added `examples/solvers/find_optimal_parameters`.
+* [Johan Karlsson](https://github.com/hilding79). Contributions to `odl.solvers`.
+* [Holger Kohr](https://github.com/adler-j). Package maintainer. Was part of the design of ODL and created several of the submodules, including the `odl.discr` and `odl.trafos` modules. Has contributed to most modules.
+* [Gregory R. Lee](https://github.com/grlee77). Bugfixes.
```

Comment:
Maybe a bit more specific, like "Bugfixes for complex spaces" or so. I don't remember the exact details.

Reply:
Rather not be so detailed here, focus is on those who made "major" contributions so to say

Reply:
I'm not thinking of intricate details, but just "Bugfixes" is a bit overdoing it focus-wise IMO. It stands out as particularly short.

Reply:
This will be hopeless to keep updated, way to detailed. Is it nit sufficient to state that these individuals have made important contributions?

Reply:
The alternative in discussion was to acknowledge the contributions in each individual release. That would at least rid us of the need to keep a file updated (except from adding names).

Reply:
The main idea for me is to let this one be community edited so to say. We fix it now, but later people will have to add themselves.

Reply:
That's actually okay for me, and if someone forgets, we can do it in the next release where we compile that information anyway. Not so much a maintenance nightmare.

Reply:
I mean it's up to them if they want the mention, it's mostly a curtesy. Their contributions are the facto listed int he commit history (thats how I wrote this).